r2/r2/controllers/oembed.py

```diff
@@ -94,9 +100,10 @@ def _oembed_post(thing, **embed_options):
                        title=websafe(thing.title),
                        subreddit_url=make_url_https(subreddit.path),
                        subreddit_name=subreddit.name,
-                       embedly_script=EMBEDLY_SCRIPT,
                        )
 
+    html += script
```

Comment:
Tiny nit: This might be a little bit cleaner if you just passed it directly into `format_html` above, as an additional param and render in the template, but not really a dealbreaker. Just maybe worth playing with.